### DISCUSSION TITLE
Unable to install v0.3-beta on Windows 10, 64-bit

### DISCUSSION THREAD

**Discussion by xawwnee** [ooflet/Mi-Create#8] [Questions-EN]
Unable to install v0.3-beta on Windows 10, 64-bit.
```
---------------------------
Error
---------------------------
This program does not support the version of Windows your computer is running.
---------------------------
OK   
---------------------------
```

**Comment by ooflet** [comment]
Can you paste in your entire Windows 10 version?

**Comment by xawwnee** [comment]
> Can you paste in your entire Windows 10 version?

Microsoft Windows [Version 10.0.17763.5696] LTSC, 64-bit.

**Reply by ooflet** [reply]
I've updated the minimum version for the 0.3 installer (https://github.com/ooflet/Mi-Create/releases/tag/v0.3), see if you can install it and if there are any compatibility issues with your version of Windows.

**Reply by xawwnee** [reply]
![image](https://github.com/ooflet/Mi-Create/assets/76479202/4d741ae1-03f2-4250-a8be-1fe2955a295b)
Same issue.

But v0.2-beta installing perfectly.
